app/components/top-nav-bar.jsx

```diff
@@ -222,6 +247,7 @@ const useStylesFromThemeFunction = createUseStyles(theme => ({
     position: 'relative',
   },
   logo: {
+    zIndex: 1,
```

Comment:
I don't think zIndex is needed here.  (I turned it off and it seemed to work the same). I'm I missing something? 
I try not to use zIndex because down the road there ends up being conflicts if there's competing zIndexes.  But if it is needed, could you add a comment about why?

Reply:
Yeah, I remember in a previous issue we discussed not using zIndex when possible. If I recall correctly, the logo wasn't appearing in the navbar for me when I tried it in Enciv Home, but the bar itself was. Then when I added zIndex it showed again. 

My thoughts were that if another component caused this, it'd block the entire bar and not just the logo. I'm not entirely sure though. 

Since the changes here weren't merged yet I copied the top-nav-bar code from this branch into `node_modules/civil-pursuit/dist/components/top-nav-bar.js`, and replaced the useStylesFromThemeFunction() part to test layout when using transparent mode in Enciv Home.

Reply:
Also, if you do need zIndex please put it in theme.zIndexes.   Also, see the one that is already there.

Reply:
> Also, see the one that is already there.

Ah, I must've missed that. I moved the theme.zIndexes.menu to apply to the entire component.